### ISSUE TITLE
update libhdfs3 git pull command

### ISSUE THREAD

**Issue by ct-clmsn** [chapel-lang/chapel#8753]
Modify this [line](https://github.com/chapel-lang/chapel/blob/cf115292ff67fed263cc958cafdce3e562e8c208/third-party/libhdfs3/fetch-libhdfs3.sh#L10) to the following command.

`svn checkout https://github.com/apache/incubator-hawq/trunk/depends/libhdfs3`

That will checkout the libhdfs3 version that was migrated from the PivotalHD GitHub repository to the Apache-Hawq repository that now hosts the maintained development tree.

**Comment by ct-clmsn** [comment]
@bradcray  Apologies! I saw @benjamin-robbins's feature request for HDF5 and my mind transposed it into HDFS.

**Comment by bradcray** [comment]
Yeah, talk about a naming fail...  But to be clear, is the modification you note still relevant/worthwhile?  Is it something you can/would file a PR against?
